src/main/java/no/nav/veilarboppfolging/repository/OppfolgingsPeriodeRepository.java

```diff
@@ -54,6 +54,13 @@ public void avslutt(String aktorId, String veileder, String begrunnelse) {
                         pageSize);
     }
 
+    public Oppfolgingsperiode hentOppfolgingsperiode(String id) {
```

Comment:
Ser riktig ut, men det er fint med en enkel test for å verifisere at det fungerer

Reply:
Har lagt til tester nå 👍